### DISCUSSION TITLE
Any way to customize the number of messages per page in the Postal web UI?

### DISCUSSION THREAD

**Discussion by JohnHammell** [postalserver/postal#3120] [Help with using Postal]
When reviewing outgoing messages there are 30 messages per page listed.

Is there a config somewhere to where I can customize the number of messages listed per page?

**Comment by willpower232** [comment]
I don't believe there is. For what its worth, Postal isn't really intended to keep messages very long so the UI can be a bit basic in this regard.

If you're keeping copies of the messages from wherever you send them then you can make use of webhooks to keep your information updated with the latest status of each message.

**Comment by JohnHammell** [comment]
I'm running Postal on a new server for handling emails sent out from a forum website also hosted on that server.

The new server IP must have been placed on a few block lists when that IP was used by previous hosting customers. So as I'm able to get the IP removed from each block list I'm going back through emails that were sent via Postal previously, but were blocked, and using the option to redeliver them.

It's very useful to be able to redeliver hardfail messages in this way with Postal. However, when there are many pages the 30 per page limit really slows down doing this manually.

It would be great if I could list 1000 messages per page, then use the find option in my browser to quickly jump to the @example.com domains to find messages that I'm ready to redeliver after cleared from the block lists.

**Reply by willpower232** [reply]
If you have a look around the issues and discussions, I think someone had some code for doing a bulk release of held messages somewhere.

**Reply by JohnHammell** [reply]
They were hardfail messages due to the IP under the previous customer's use being blocked by a few providers. I was able to get that all cleared up, though, with those few email providers fairly quickly. Around 1600 emails were in hardfail status and I went through all of them today, just over 50 pages of emails listed. Redelivered all that were previously IP blocked. Ended up being 500 emails, approx that were able to be successfully redelivered.
